lib/protobuf/any.ex

```diff
@@ -0,0 +1,37 @@
+defmodule Protobuf.Any do
+  @type_url_prefix "type.googleapis.com/"
+
+  def pack(%mod{} = data) do
+    Google.Protobuf.Any.new(%{
+      type_url: "#{@type_url_prefix}#{mod.full_name()}",
+      value: mod.encode(data)
+    })
+  end
+
+  def unpack(
+        %{__struct__: Google.Protobuf.Any, type_url: @type_url_prefix <> name, value: value},
+        options \\ []
+      ) do
+    prefix = Keyword.get(options, :prefix, nil)
+
+    parts =
+      name
+      |> String.split(".")
+      |> Enum.map(fn x ->
+        if String.match?(x, ~r/A-Z/) do
+          x
+        else
+          Macro.camelize(x)
+        end
+      end)
+
+    mod =
+      if prefix do
+        Module.concat([prefix] ++ parts)
+      else
+        Module.concat(parts)
```

Comment:
We should be careful about using `Module.concat` since it creates new atoms this means we can only unpack from trusted data sources which I am not sure we should limit ourselves to. I think we need another solution for this.

Reply:
I've updated this to use safe_concat.... Would that take care of the issue?

Reply:
Yes, just make sure to mention this when you add the documentation.